perfrunbook/utilities/find_and_list_jar_with_so.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [[ $(id -u) -ne 0 ]]; then
```

Comment:
really, why?

Reply:
The find command searches the entire disk.  Without sudo, it errors out. I don't want to assume the machine this gets run on deploys artifacts into home, or only to some canned directory.  There are a multitude of places JARs may get placed.

Reply:
i'd expect most to be readable